kasa/tests/fixtures/KL125(US)_1.20_1.0.7.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
This could be added in a separate PR as it's nice to have clean PRs, but it's fine also as it is.

Reply:
Good point - this is somewhat unrelated;  yanking and will submit via another PR later.